js/src/Map.js

```diff
@@ -56,7 +56,7 @@ var LeafletMapModel = widgets.DOMWidgetModel.extend({
         inertia_deceleration : 3000,
```

Comment:
When you actually create the Leaflet map (line 201 of this file), you should pass `zoom_control: false` to the Map constructor, so that Leaflet does not create a zoom control for us.